package.json

```diff
@@ -52,6 +52,7 @@
     }
   },
   "dependencies": {
+    "@babel/helper-module-imports": "7.12.5",
```

Comment:
Pinning an exact version (without a `^`) is 100% an anti-pattern for a published package, but I know from past discussions your mind can't be changed so I made it consistent with the other dependencies.

Reply:
Note that by pinning an exact version, if a user already has a newer patch than `7.12.5` in their `node_modules`, installing `styled-jsx` will downgrade it, or create duplicate installations of `@babel/helper-module-imports` at different versions within `node_modules`. Nasty.